source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -61,14 +62,39 @@
                 }
             });
         } else {
-            //TODO: show alert popup
-            alert("HAY CALIFICACIONES INVÁLIDAS");
+            var modalContainer = $('#evaluations-generated-confirm');
```

Comment:
Great! I think that @joseguti84 already did something similar, let me find the code.